### ISSUE TITLE
Current git head of moksha unconditionally checks for pulse

### ISSUE THREAD

**Issue by TommyD** [JeffHoogland/moksha#80]
As far as i can see, pulseaudio is only used together with the mixer module. But the configure check for the pulseaudio libraries are not conditionally based on enabled mixer module.

So even if the mixer module is disabled and there is no need for pulseaudio, current git head of moksha requires pulseaudio to be installed. Please make this configure.ac check conditional based on the check, if mixer module is enabled or not.

**Comment by rbtylee** [comment]
Yes you are right. I was aware of this. My goal right now has been to get Moksha 3.0 ready for the release of Bodhi 5.0. Testing configure.ac for all possible options to ensure they all work correctly is low priority at this moment, it is on my list of stuff to do eventually. Until then patches are accepted if they work , barring that,  unless I can interest someone else on working on this it is going to have to wait.

Thanks for reporting the issue tho. It will ensure we don't neglect it.

**Comment by rbtylee** [comment]
commit [conditionally check for pulse in Mixer module]( https://github.com/JeffHoogland/moksha/commit/f1e81898a739a4e434fff44954f6eeaa12eceb75) addresses this issue. Sorry for the long delay but I have been messing with the configure.ac file for other reasons soon to be committed and went ahead and fixed this.

I am closing this issue. However, I have not tested this with pulse not installed so if it fails to work as anticipated please reopen this issue
